Source/DirectoryWatcher.swift

```diff
@@ -86,30 +93,45 @@ public class DirectoryWatcher {
     self.dispatchSource = nil
   }
   
-  func handleChangeEvent() {
+  func getCurrentFiles() -> [URL] {
+    do {
+      return try FileManager.default.contentsOfDirectory(
+        at: url,
+        includingPropertiesForKeys: [.creationDateKey, .typeIdentifierKey],
+        options: [.skipsHiddenFiles]
+      )
+    } catch {
+      delegate?.directoryWatcher(self, error: error)
+    }
+    return []
+  }
```

Comment:
@dworak, in my opinion, it will look better if you put `return []` inside `catch` block so both `return` happen inside ` do / catch `expression. It will look more consistent.

```
 func getCurrentFiles() -> [URL] {
    do {
      return try FileManager.default.contentsOfDirectory(
        at: url,
        includingPropertiesForKeys: [.creationDateKey, .typeIdentifierKey],
        options: [.skipsHiddenFiles]
      )
    } catch {
      delegate?.directoryWatcher(self, error: error)
      return []
    }
  }
```